config/clients/java/template/OpenFgaApiTest.java.mustache

```diff
@@ -0,0 +1,709 @@
+                "Missing the required parameter 'authorizationModelId' when calling readAssertions",
+                exception.getMessage());
+    }
+
+    /**
+     * Return a particular version of an authorization model.
+     */
+    @Test
+    public void readAuthorizationModelTest() throws ApiException {
+        // Given
+        mockHttpClient
+                .onGet(
+                        "https://localhost/stores/01YCP46JKYM8FJCQ37NMBYHE5X/authorization-models/01G5JAVJ41T49E9TT3SKVS7X1J")
+                .doReturn(
+                        200,
+                        String.format(
+                                "{\"authorization_model\":{\"id\":\"%s\",\"schema_version\":\"%s\"}}",
+                                DEFAULT_AUTH_MODEL_ID, DEFAULT_SCHEMA_VERSION));
+
+        // When
+        ReadAuthorizationModelResponse response = fga.readAuthorizationModel(DEFAULT_STORE_ID, DEFAULT_AUTH_MODEL_ID);
+
+        // Then
+        mockHttpClient
+                .verify()
+                .get(
+                        "https://localhost/stores/01YCP46JKYM8FJCQ37NMBYHE5X/authorization-models/01G5JAVJ41T49E9TT3SKVS7X1J")
+                .called(1);
+        assertNotNull(response.getAuthorizationModel());
+        assertEquals(DEFAULT_AUTH_MODEL_ID, response.getAuthorizationModel().getId());
+        assertEquals(DEFAULT_SCHEMA_VERSION, response.getAuthorizationModel().getSchemaVersion());
+    }
+
+    @Test
+    public void readAuthorizationModel_storeIdRequired() throws ApiException {
+        // When
+        ApiException exception =
+                assertThrows(ApiException.class, () -> fga.readAuthorizationModel(null, DEFAULT_AUTH_MODEL_ID));
+
+        // Then
+        assertEquals(
+                "Missing the required parameter 'storeId' when calling readAuthorizationModel", exception.getMessage());
+    }
+
+    @Test
+    public void readAuthorizationModel_idRequired() throws ApiException {
+        // When
+        ApiException exception =
+                assertThrows(ApiException.class, () -> fga.readAuthorizationModel(DEFAULT_STORE_ID, null));
+
+        // Then
+        assertEquals("Missing the required parameter 'id' when calling readAuthorizationModel", exception.getMessage());
+    }
+
+    /**
+     * Return all the authorization models for a particular store.
+     */
+    @Test
+    public void readAuthorizationModelsTest() throws ApiException {
+        // Given
+        mockHttpClient
+                .onGet("https://localhost/stores/01YCP46JKYM8FJCQ37NMBYHE5X/authorization-models")
+                .doReturn(
+                        200,
+                        String.format(
+                                "{\"authorization_models\":[{\"id\":\"%s\",\"schema_version\":\"%s\"}]}",
+                                DEFAULT_AUTH_MODEL_ID, DEFAULT_SCHEMA_VERSION));
+
+        // When
+        ReadAuthorizationModelsResponse response = fga.readAuthorizationModels(DEFAULT_STORE_ID, null, null);
```

Comment:
curious what are the parameters for readAuthorizationModels?

Reply:
[2d5c18f](https://github.com/openfga/sdk-generator/pull/170/commits/2d5c18f071091041bc3898c26fb276e77b91171a)